Sources/PixelKit/PixelKit+Parameters.swift

```diff
@@ -54,6 +54,7 @@ public extension PixelKit {
         public static let latency = "latency"
         public static let server = "server"
         public static let networkType = "net_type"
+        public static let domain = "domain"
```

Comment:
Simple addition of a domain parameter, since this is quite a generic concept that could be reused.